docs/services.md

```diff
@@ -658,7 +658,7 @@ const credentials = {
   url: '{s3 url}'
 };
 uploadsClient.createUpload({
-  titleset: `${myUsername}.${myTileset}`,
+  tileset: `${myUsername}.${myTileset}`,
   url: credentials.url,
   name: 'my uploads name',
   private: true
```

Comment:
the `private` property throws another error:

`message: 'The request body contains the invalid property "private".'`

Reply:
That's probably #392

Reply:
@andrewharvey yep, I think there is a way to satisfy the spec by not setting default value for private on the client, anyway left a comment in #392